### ISSUE TITLE
Independent CSS

### ISSUE THREAD

**Issue by darthtrevino** [adazzle/react-data-grid#441]
**I'm submitting a ...**  (check one with "x")

```
[ ] bug report
[X] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/adazzle/react-data-grid/blob/master/CONTRIBUTING.md
```

**Current behavior**
I couldn't figure out why my grid was rendering incorrectly. It turns out it's because I'm not including Bootstrap CSS. 

**Expected/desired behavior**
Ideally, the styling for the data grid would not depend on a CSS framework. I would recommend developing sans Bootstrap, or at the very least having a barebones test-bed. At the very least, the documentation should be updated to call Bootstrap out as a styling dependency.

**Reproduction of the problem**
Render a basic grid without Bootstrap CSS. 

**What is the expected behavior?**
The grid matches the rendered grids in the documentation page.

**What is the motivation / use case for changing the behavior?**
Greater stylistic freedom; complete dependency capturing

**Comment by tg-adz** [comment]
Agreed @darthtrevino we should produce a single css with all relevant styles.

**Comment by willemx** [comment]
Why is this issue closed ?
It has not been solved yet, see #634

**Comment by jpdriver** [comment]
hi @willemx -- this was probably closed because styles should be included out of the box as of 2.0.

it looks like there might be some problems with this feature though -- i've only just seen [this comment](https://github.com/adazzle/react-data-grid/issues/625#issuecomment-281732250) -- which is obviously something we want to discourage as strongly as possible.

@diogofcunha is Adazzle aware of the issues?

**Comment by ChadRehmKineticData** [comment]
FYI: react-data-grid 3.0.11 has a dependency on bootstrap when resizable or draggable is implemented.  It wasn't in the setup instructions.
